model/task/task.go

```diff
@@ -1273,6 +1260,35 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 	return nil
 }
 
+// GetTaskIdBetweenIds gets the task between two task given that they are
+// from the same project, requester, build variant, and display name. The
+// order of the ID's does not matter and if the task passed cannot have a
+// middle (i.e. it is sequential tasks or the same task) it will return the
+// the first task given.
+func GetMidwayTask(t1, t2 Task) (*Task, error) {
```

Comment:
nit: it's slightly more conventional in Evergreen for helper functions wrapping find queries to use `Find` instead of `Get` in the name.